namespace cpp database.protocol

// type: STORAGE_ACTOR_LOGOUT_CENTER
struct StorageActorLogoutCenterRequest {
  1 : required i64 id_;           // 角色ID
}

struct StorageActorLogoutCenterResponse {
}

